server/lib/update-linked-fields.js

```diff
@@ -5,7 +5,7 @@ import Donor from '../models/donor'
 /**
 * Update Volunteer fields of address
 */
-export async function updateVolunteer(reqbodyfields, user_id) {
+ async function updateVolunteer(reqbodyfields, user_id) {
```

Comment:
it's almost the same function repeated 3 times. if you pass in the client type you can combine them into one function. (and maybe move the for loops out into a helper function)